release/README.md

```diff
@@ -8,40 +8,53 @@ https://semver.org/
 1. To release a major or minor version create a release branch `release-X.Y` from the last minor release tag.
 
     ```bash
-    git checkout vA.B.0
+    git checkout main
```

Comment:
Should we really branch from `main`? I think the idea behind branching from the previous release is to be able to determine what changed between them. The release is then a PR that will always tell you exactly what changed and you can check every commit if you'd like. If we start from `main` there is no such PR, just a tag that you can diff with the last tag.

The issue I see with this is that fixes we add during QA could be "doubled", but maybe I'm missing something?
I mean we add it during QA to the release, but then we also merge it to main, so the next time we branch from main that fix will be there again, including the note about the fix that was already in the last release.

I'm confusing myself :sweat_smile: but here is an attempt to describe it:

1. v1.0 is released and we start working on the next version on main, adding changes to the WIP change log.
2. We start preparing for v1.1 by branching from main and clearing the WIP change log.
3. We add a fix to the soon-to-be release that also requires a change to the (WIP?) change log.
4. Release v1.1
5. Merge the fix back to main, adding the change log note
6. Start working on v1.2. Now the change log contains the note about that fix that was already in the last release.

Reply:
I don't really see the difference other than in one you get the diff in the PR the other you need to run `git diff ${PREV_VERSION}` the history should be exactly the same in the end.

Maybe we should have QA fixes into WIP changelog. But I thought we should add them to the CHANGELOG directly (since they are part of the release. Then when we merge the fixes they should end up directly in the changelog and nothing gets wierd.
Because otherwise, if we have it like we have it now. When we merge into main we will always have a conflict in WIP if there's more work added since the branching and you need to manually figure out what has been added and not. I just feel like this flow would produce less merge conflicts.

Reply:
> But I thought we should add them to the CHANGELOG directly (since they are part of the release. Then when we merge the fixes they should end up directly in the changelog and nothing gets wierd.

Ah this was the part I was missing and a really nice solution! Could you mention it maybe in the document as well? Otherwise I'm sure I'll add something in the wrong change log. :smile: